src/components/overview/helpers/productAPI.js

```diff
@@ -0,0 +1,36 @@
+// eslint-disable-next-line import/newline-after-import
+/* eslint-disable no-console */
+const axios = require('axios');
+const config = require('../../../config.js');
+
+const instance = axios.create({
+  baseURL: 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/',
+  headers: {
+    Authorization: config.API_TOKEN,
+  },
+});
+
+// function getStyles(productId) {
+//   return instance.get(`/${productId}/styles`)
+//     .then((res) => {
+//       console.log('Here are my styles:', res.data);
+//     })
+//     .catch((err) => {
+//       Error('Error with productAPI getStyle', err);
+//     });
+// }
+
+function getProducts() {
+  console.log('I am here in API');
+  return instance.get('/products')
+    .then((res) => {
+      console.log('Here is my data: ', res.data);
+      res.status(200).send(res);
```

Comment:
Because `res` in this scope refers to the data we get back from the API, we can't do `res.status()` or `res.send()`. Instead, we can just do something like `return res.data` to send the data back to the server. If you're still getting a 401 error, you might have to generate a new API token?